azure-devops/azext_devops/test/team/test_team.py

```diff
@@ -0,0 +1,126 @@
+        patch.stopall()
+
+    def test_create_team(self):
+        create_team(self._TEST_TEAM_NAME, self._TEST_TEAM_DESCRIPTION, self._TEST_DEVOPS_ORGANIZATION, self._TEST_PROJECT_NAME, 'Off')
+        
+        #assert
+        self.mock_create_team.assert_called_once()
+        create_team_param = self.mock_create_team.call_args_list[0][1]
+        self.assertEqual(self._TEST_PROJECT_NAME, create_team_param['project_id'], str(create_team_param))
+        self.assertEqual(self._TEST_TEAM_NAME, create_team_param['team'].name, str(create_team_param))
+        self.assertEqual(self._TEST_TEAM_DESCRIPTION, create_team_param['team'].description, str(create_team_param))
+
+    def test_delete_team(self):
+        delete_team(self._TEST_TEAM_NAME, self._TEST_DEVOPS_ORGANIZATION, self._TEST_PROJECT_NAME, 'Off')
+
+        #assert
+        self.mock_delete_team.assert_called_once()
+        delete_team_param = self.mock_delete_team.call_args_list[0][1]
+        self.assertEqual(self._TEST_PROJECT_NAME, delete_team_param['project_id'], str(delete_team_param))
+        self.assertEqual(self._TEST_TEAM_NAME, delete_team_param['team_id'], str(delete_team_param))
+
+    def test_get_team(self):
+        get_team(self._TEST_TEAM_NAME, self._TEST_DEVOPS_ORGANIZATION, self._TEST_PROJECT_NAME, 'Off')
+        
+        #assert
+        self.mock_get_team.assert_called_once()
+        get_team_param = self.mock_get_team.call_args_list[0][1]
+        self.assertEqual(self._TEST_PROJECT_NAME, get_team_param['project_id'], str(get_team_param))
+        self.assertEqual(self._TEST_TEAM_NAME, get_team_param['team_id'], str(get_team_param))
+    
+    def test_get_teams(self):
+        get_teams(self._TOP_VALUE, self._SKIP_VALUE, self._TEST_DEVOPS_ORGANIZATION, self._TEST_PROJECT_NAME, 'Off')
+
+        #assert
+        self.mock_get_teams.assert_called_once()
+        get_teams_param = self.mock_get_teams.call_args_list[0][1]
+        self.assertEqual(self._TEST_PROJECT_NAME, get_teams_param['project_id'], str(get_teams_param))
+        self.assertEqual(10, get_teams_param['top'], str(get_teams_param))
+        self.assertEqual(2, get_teams_param['skip'], str(get_teams_param))
+
+    def test_get_team_members(self):
+        get_team_members(self._TEST_TEAM_NAME, self._TOP_VALUE, self._SKIP_VALUE, self._TEST_DEVOPS_ORGANIZATION, self._TEST_PROJECT_NAME, 'Off')
+
+        #assert
+        self.mock_get_team_members.assert_called_once()
+        get_team_members_param = self.mock_get_team_members.call_args_list[0][1]
+        self.assertEqual(self._TEST_TEAM_NAME, get_team_members_param['team_id'], str(get_team_members_param))
+        self.assertEqual(self._TEST_PROJECT_NAME, get_team_members_param['project_id'], str(get_team_members_param))
+        self.assertEqual(10, get_team_members_param['top'], str(get_team_members_param))
+        self.assertEqual(2, get_team_members_param['skip'], str(get_team_members_param))
+
+    def test_update_team(self):
```

Comment:
Can we add a test case for update_team where both name and description are none?

Reply:
Added a test with name and description both none.